evm/src/cpu/kernel/asm/rlp/read_to_memory.asm

```diff
@@ -13,29 +13,26 @@ global read_rlp_to_memory:
 
     PUSH @SEGMENT_RLP_RAW // ctx == virt == 0
     // stack: addr, final_addr, retdest
-
 read_rlp_to_memory_loop:
     // stack: addr, final_addr, retdest
     DUP2
     DUP2
-    EQ
-    // stack: addr == final_addr, addr, final_addr, retdest
+    LT
+    ISZERO
+    // stack: addr >= final_addr, addr, final_addr, retdest
```

Comment:
can't we just use `GE`?

Reply:
Nevermind, I saw 2 macros using GE so I thought we could use it, but it actually doesn't exist 🤦🏼 
So looks good! (the macros are never used hence why it didn't trigger compile errors)